modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_topper.scss

```diff
@@ -37,6 +49,11 @@ html#{$cadmin-selector} {
 			display: inline-block;
 		}
 
+		.portlet-title-default > .portlet-name-text {
```

Comment:
The style changes here could go in https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_topper.scss#L67

Reply:
@pat270 thanks for the feedback! I applied both changes 😇 let me know if everything is ok now

Reply:
LGTM!